src/patternfly/components/Panel/panel.scss

```diff
@@ -1,53 +1,59 @@
 
   // bordered
-  --#{$panel}--m-bordered--before--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
+  --#{$panel}--m-bordered--before--BorderWidth: var(--pf-t--global--border--width--regular);
 
   // raised
-  --#{$panel}--m-raised--BoxShadow: var(--#{$pf-global}--BoxShadow--md);
-  --#{$panel}--m-raised--ZIndex: var(--#{$pf-global}--ZIndex--sm);
-  --#{$panel}--m-raised--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$panel}--m-raised--BoxShadow: var(--pf-t--global--box-shadow--md);
+  --#{$panel}--m-raised--ZIndex: var(--pf-t--global--Zindex--sm);
+  --#{$panel}--m-raised--BackgroundColor: var(--pf-t--global--background--color--primary--default);
 
   // header
-  --#{$panel}__header--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__header--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__header--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__header--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$panel}__header--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$panel}__header--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$panel}__header--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$panel}__header--PaddingLeft: var(--pf-t--global--spacer--md);
 
   // main
   --#{$panel}__main--MaxHeight: none;
   --#{$panel}__main--Overflow: visible;
 
   // body
-  --#{$panel}__main-body--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__main-body--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__main-body--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__main-body--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$panel}__main-body--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$panel}__main-body--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$panel}__main-body--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$panel}__main-body--PaddingLeft: var(--pf-t--global--spacer--md);
 
   // footer
-  --#{$panel}__footer--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__footer--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__footer--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$panel}__footer--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$panel}__footer--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$panel}__footer--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$panel}__footer--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$panel}__footer--PaddingLeft: var(--pf-t--global--spacer--md);
   --#{$panel}__footer--BoxShadow: none;
 
   // scrollable
   --#{$panel}--m-scrollable__main--MaxHeight: #{pf-size-prem(300px)};
   --#{$panel}--m-scrollable__main--Overflow: auto;
   --#{$panel}--m-scrollable__footer--BoxShadow: 0 #{pf-size-prem(-5px)} #{pf-size-prem(4px)} #{pf-size-prem(-4px)} #{rgba($pf-v5-color-black-1000, .16)}; // insets the shadow so it doesn't show on left/right sides
```

Comment:
Is there a token we want to use for this hardcoded rgba?

Reply:
I'm not sure we'll even want a box-shadow there? If so, @andrew-ronaldson do you want to pick a palette color to replace it since this color will go away, and then @thatblindgeye we can put in a todo or follow up issue so we don't lose this?

Reply:
fwiw, with our [box shadow tokens](https://github.com/patternfly/patternfly/blob/d244d9f50027a16269548cc204b4e6edfc162dac/src/patternfly/base/tokens/_tokens-font.scss#L59-L73) we're just using black (`rgba(0 0 0 / 16%)`)